packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -1,25 +1,48 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
 import '../src/index.ts';
+import { getDisplayCode } from '../../../../../.storybook/common';
 
-const autocompleteDataReceivedEvent = new CustomEvent('autocomplete_received_results',
+// @TODO allow for sending event with searchbar ID. This should allow for one
+// story's events to not affect another story.
+const autocompleteDataReceivedEvent = new CustomEvent('sfx::autocomplete_received_results',
   { detail: [
       { "title": "Brands", "items": [{ "label": "Cats" }, { "label": "Dogs" }] },
       { "title": "default", "items": [{ "label": "Cars" }, { "label": "Bikes" }] }
```

Comment:
The autocomplete component functions on the title property being empty rather than the string 'default'
{ "title": "", "items": [{ "label": "Cars" }, { "label": "Bikes" }] }
instead of 
{ "title": "default", "items": [{ "label": "Cars" }, { "label": "Bikes" }] }